### PR TITLE
Update to add English Level support to FE, BE, and DB.

### PR DESCRIPTION
**Pull request by rhodesconover1**
This change set addresses issue 23.

### CODE REVIEW

File: Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs [hunk 1]
```diff
@@ -58,6 +58,19 @@ public async Task SetDisability([FromBody]KeyValuePair<int?, string> disability,
         await repository.SetDisability(disability.Key, disability.Value, outreachTeamId);
     }
 
+
+    [HttpGet("english-levels")]
+    public async Task<Dictionary<int, string>> GetEnglishLevelLookup([FromQuery(Name = "otid")]int outreachTeamId)
```
Comment (Tolkiennerd) [inline Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs]: Could this method be renamed to the plural? `GetEnglishLevelsLookup`

File: Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs [hunk 2]
```diff
@@ -58,6 +58,19 @@ public async Task SetDisability([FromBody]KeyValuePair<int?, string> disability,
...
+    {
+        return await repository.GetEnglishLevel(outreachTeamId);
```
Comment (Tolkiennerd) [inline Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs]: Plural: `GetEnglishLevels`

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs [hunk 3]
```diff
@@ -5,6 +5,7 @@ internal static class LookupsMySqlQueries
     internal static readonly string GetCushionConditions = "SELECT CushionConditionId, CushionCondition FROM CushionCondition WHERE OutreachTeamId=@OutreachTeamId";
     internal static readonly string GetCushionTypes = "SELECT CushionTypeId, CushionType FROM CushionType WHERE OutreachTeamId=@OutreachTeamId";
     internal static readonly string GetDisabilities = "SELECT DisabilityId, Disability FROM Disability WHERE OutreachTeamId=@OutreachTeamId";
+    internal static readonly string GetEnglishLevel = "SELECT EnglishLevelId, EnglishLevel FROM EnglishLevel WHERE OutreachTeamID=@OutreachTeamId";
```
Comment (Tolkiennerd) [inline Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs]: Plural: `GetEnglishLevels`

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs [hunk 4]
```diff
@@ -21,6 +22,7 @@ internal static class LookupsMySqlQueries
         GetCushionConditions,
         GetCushionTypes,
         GetDisabilities,
+        GetEnglishLevel,
```
Comment (Tolkiennerd) [inline Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs]: Plural

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlRepository.cs [hunk 5]
```diff
@@ -20,6 +20,11 @@ public async Task<Dictionary<int, string>> GetDisabilities(int outreachTeamId)
         return await GetLookup(LookupsMySqlQueries.GetDisabilities, outreachTeamId);
     }
 
+    public async Task<Dictionary<int, string>> GetEnglishLevel(int outreachTeamId)
```
Comment (Tolkiennerd) [inline Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlRepository.cs]: Plural

**Review by Tolkiennerd — CHANGES_REQUESTED** [review]

File: Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs (hunk 2 above)
Comment (rhodesconover1) [reply Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs]: Fixed.

File: Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs (hunk 1 above)
Comment (rhodesconover1) [reply Backend/UnhousedOutreach.Api/Controllers/LookupsController.cs]: Fixed.

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs (hunk 3 above)
Comment (rhodesconover1) [reply Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs]: Fixed.

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs (hunk 4 above)
Comment (rhodesconover1) [reply Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlQueries.cs]: Fixed.

File: Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlRepository.cs (hunk 5 above)
Comment (rhodesconover1) [reply Backend/UnhousedOutreach.Database/Lookups/LookupsMySqlRepository.cs]: Fixed.

**Review by Tolkiennerd — APPROVED** [review]

**Pull request merged**